src/linter.js

```diff
@@ -98,19 +102,42 @@ export default class Linter {
   }
 
   handleError(err, _console = console) {
-    if (err.message.includes('DuplicateZipEntry')) {
+    // The zip files contains invalid entries (likely path names using invalid
+    // characters like '\\'), the linter can inspect the package but Firefox
+    // would fail to load it.
+    if (err instanceof InvalidZipFileError) {
+      this.collector.addError({
+        ...messages.INVALID_XPI_ENTRY,
```

Comment:
I do wonder if we should rename the exception in addons-scanner-utils now :D

Reply:
Honestly I was also tempted to propose you that, but in the end from the linter perspective the fact that the rule name doesn't exactly match the addons-scanner-utils error name is annoying but probably not "annoying enough" to justify releasing a new addons-scanner-utils version just to rename the error class.